### PR TITLE
Prepare for Eclipse Marketplace release

### PR DESCRIPTION
**Pull request by arafsheikh**
Prepare plugin for Eclipse Marketplace release

### CODE REVIEW

**Comment by sils** [comment]
@arafsheikh you sure we should have jar files in here? Shouldn't we like generate them and upload them somewhere? Maybe @jayvdb knows a solution.

**Comment by arafsheikh** [comment]
Yes sure we should do that.

File: README.md [hunk 1]
```diff
@@ -24,3 +22,42 @@ mvn clean verify
...
+
+3. In the `Run coala with` push menu, select the the bear you wish to run.
+
+4. The plug-in will then run the analysis and display the results in the `Problems` pane.
+
+### Fixing issues returned after analysis
+
+1. After the analysis is complete and the issues are visible in the `Problems` pane, right-click an issue and select the `Quick Fix` option (*Ctrl +1* keyboard shortcut).
+
+2. In the `Quick Fix` dialog select the fix that you want to apply and click `Finish`.
```
Comment (hemangsk) [inline README.md]: A comma will be better here
In the `Quick Fix` dialog, select the fix that you want to apply and click `Finish`

**Comment by sils** [comment]
@arafsheikh what's the plan on continuing this? Do you need anything? Marking WIP for now. You should at least fix @hemangsk s comment, if possible resolve the jar issue though I guess merging this without resolving that is probably better than not merging it.

**Comment by arafsheikh** [comment]
Yeah let's merge this. I'll open an issue regarding the jar files.

**Comment by sils** [comment]
@arafsheikh ok merge it, this is your repository. But please, can we get this plugin to a stable state soon?

**Comment by arafsheikh** [comment]
@sils What do you mean by stable state?

**Comment by sils** [comment]
@arafsheikh well a state where we have no such things as jar files that we don't want and no such things as PRs that would be usually rejected but we have to merge them because the plugin is broken otherwise. I assume the plugin is working now?

Generally stability of the plugins is a huge issue because nobody finds the time to update them when we do a release so things like this happen (coala-json not working anymore e.g.)

**Comment by arafsheikh** [comment]
@sils The install/update is still causing some issue. It is picking up an older jar file to install from. Working on it. Once this is fixed I guess the plugin will be usable.

**Pull request merged**